modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -1624,12 +1624,15 @@ export class Wallet implements IWallet {
 
     const signTransactionParams = {
       ...presign,
-      txPrebuild,
+      txPrebuild: { ...txPrebuild, walletId: this.id() },
       pubs,
       coin: this.baseCoin,
     };
 
     if (_.isFunction(params.customSigningFunction)) {
+      if (typeof this.baseCoin.signWithCustomSigningFunction === 'function') {
+        return await this.baseCoin.signWithCustomSigningFunction(params.customSigningFunction, signTransactionParams);
```

Comment:
```suggestion
        return this.baseCoin.signWithCustomSigningFunction(params.customSigningFunction, signTransactionParams);
```
nit: unneeded await.

Reply:
Done